packages/app/pages/__meetings/[id]/index.tsx

```diff
@@ -30,38 +30,55 @@ import {
 } from "../../../src/ui";
 import { CheckSquare, Edit } from "../../../src/ui/icons";
 
+/**
+ * @deprecated
+ */
```

Comment:
This entire page is redesign explosion fallout.
I considered removing it, but I want to keep it working, because we'll enable it in the next milestone.

Reply:
I see, thanks for clarifying!